tests/unit/youtube/wrapper-api-tests.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * https://github.com/Automattic/expect.js
+ */
+
+describe('youtube wrapper tests', function () {
+  // Load test helpers
+  require('../../utils/test_lib.js');
+  jest.dontMock('../../utils/mock_vtc.js');
+  require('../../utils/mock_vtc.js');
+
+  var pluginFactory, parentElement, wrapper, element, vtc, originalTimeout;
+
+  // Setup
+  OO.Video = { plugin: function(plugin) { pluginFactory = plugin; } };
+
+  // Load file under test
+  jest.dontMock('../../../src/youtube/js/youtube');
+  require('../../../src/youtube/js/youtube');
+
+  beforeEach(function() {
+    vtc = new mock_vtc();
+    parentElement = $("<div>");
+    wrapper = pluginFactory.create(parentElement, "test", vtc.interface, {});
+    element = parentElement.children()[0];
+    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 3000;
+  });
+
+  afterEach(function() {
+    OO.isEdge = false;
+    OO.isAndroid = false;
```

Comment:
Should this go into beforeEach and after (not afterEach)? This will ensure the first test has the proper values instead of what remains from previous test cases.